app/models/user.rb

```diff
@@ -0,0 +1,21 @@
+class User
+  include DataMapper::Resource
+
+  has n, :posts, through: Resource
+
+  property :id,       Serial
+  property :username, String
+  property :name,     String
+  property :email,    Text, :format => :email_address
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)